db/migrate/20210302124316_create_reviews.rb

```diff
@@ -1,7 +1,9 @@
 class CreateReviews < ActiveRecord::Migration[6.1]
   def change
     create_table :reviews do |t|
-
+      t.string  :text
+      t.integer :count
+      t.float :star
```

Comment:
こちらも、NOT NULL制約をつけましょう！

```suggestion
      t.string  :text, null: false
      t.integer :count, null: false
      t.float :star, null: false
```